Sync `css/css-transforms` from WPT upstream

Ahmad-S792 opened the pull request:
#### e25d35159393345f3d9891b9c6229db11280d397
<pre>
Sync `css/css-transforms` from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=294837">https://bugs.webkit.org/show_bug.cgi?id=294837</a>
<a href="https://rdar.apple.com/154088766">rdar://154088766</a>

Reviewed by Tim Nguyen.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/a572d078b6d2631443d47551c4b5780554b9654f">https://github.com/web-platform-tests/wpt/commit/a572d078b6d2631443d47551c4b5780554b9654f</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/animation/rotate-transform-equivalent.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/backface-visibility-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/backface-visibility-hidden-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-group-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-nested-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-nested-014.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-nested-019.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/parsing/transform-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/parsing/transform-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/perspective-transforms-equivalence.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-and-flattening-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewX/svg-skewxy-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-006.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-016.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-021.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-box/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-compound-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-compound-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-compound-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-input-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-percent-008.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform3d-sorting-006.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-2-transforminterop-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-2-transforminterop.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-3-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-3.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-coplanar-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-coplanar.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-overlapping-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-overlapping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/css-scale-of-clip-path-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/css-scale-of-clip-path-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/css-scale-of-clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/individual-transform-3-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/individual-transform-3.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-002-expected.xht
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-003-expected.xht
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-004-expected.xht
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-1-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-1.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-animation-with-var-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-animation-with-var-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-transform-filtered-text-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-transform-filtered-text-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-transform-filtered-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-box/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-matrix-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-matrix-009.html:

&gt; Platform Specific Expectations:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/296648@main">https://commits.webkit.org/296648@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fe5f3e3c7e072ee4bc5d65f50acf31402435dc89

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/109190 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/28848 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/19275 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/114397 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/59465 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/29529 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/37437 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/82974 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/112138 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/23497 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/98343 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/63423 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/22888 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/16486 "Passed tests") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/59041 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/92864 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/16528 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/117513 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/36234 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/26805 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/91992 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/36606 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/94606 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/91798 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23374 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/36719 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/14469 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/32078 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/36130 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/35816 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/39150 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/37507 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/perspective-transforms-equivalence.html:
```diff
@@ -9,7 +9,7 @@
 pixel differences, so the test should fuzzy patch to the ref.
 -->
-<meta name="fuzzy" content="maxDifference=0-178;totalPixels=0-538">
+<meta name="fuzzy" content="maxDifference=0-94;totalPixels=0-538">
```
This undoes this unexported change: https://github.com/WebKit/WebKit/commit/49ef3002488aa8b83b2f1975816d4172863f1554 from @nikolaszimmermann

Ahmad-S792 replied:
I will undo it. :-) Plus add any other test expectations in next commit.

nikolaszimmermann replied:
@nt1m - indeed, it is unfortunate. I should have exported this to WPT?

nt1m replied:
Yes, unfortunarely there's no auto-export in WebKit, you have to go through the `export-w3c-test-changes` script, this command should do it:

`export-w3c-test-changes -b 290800 -g 49ef3002488aa8b83b2f1975816d4172863f1554 -c`

nikolaszimmermann replied:
Thanks a lot, I did not know about the script, did it manually last time I had to do it. That's going to simplify a lot 👌

nt1m replied:
Ahmad exported them here: https://github.com/web-platform-tests/wpt/pull/53370

Ahmad-S792 committed:
Sync `css/css-transforms` from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=294837
rdar://154088766

Reviewed by Tim Nguyen.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/a572d078b6d2631443d47551c4b5780554b9654f

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/animation/rotate-transform-equivalent.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/backface-visibility-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/backface-visibility-hidden-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-group-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-nested-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-nested-014.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/group/svg-transform-nested-019.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/parsing/transform-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/parsing/transform-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/perspective-transforms-equivalence.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-and-flattening-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewX/svg-skewxy-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-006.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-016.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/skewY/svg-skewy-021.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-box/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-compound-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-compound-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-compound-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-input-005.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-percent-008.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform3d-sorting-006.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-2-transforminterop-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-2-transforminterop.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-3-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-3.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-coplanar-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-coplanar.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-overlapping-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping-overlapping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/3d-point-mapping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/css-scale-of-clip-path-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/css-scale-of-clip-path-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/css-scale-of-clip-path.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/individual-transform-3-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/individual-transform-3.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-002-expected.xht
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-003-expected.xht
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-004-expected.xht
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/individual-transform/stacking-context-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-1-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-1.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-2-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/preserve3d-backface-hit-test-2.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-animation-with-var-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-animation-with-var-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-transform-filtered-text-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-transform-filtered-text-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/scale-transform-filtered-text.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-box/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-matrix-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/transform-matrix-009.html:

> Platform Specific Expectations:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:

Canonical link: https://commits.webkit.org/296648@main
